src/Stratis.Bitcoin.Features.Interop/Payloads/ConversionRequestStatePayload.cs

```diff
@@ -0,0 +1,71 @@
+﻿using NBitcoin;
+using Stratis.Bitcoin.P2P.Protocol.Payloads;
+using Stratis.Features.FederatedPeg.Conversion;
+
+namespace Stratis.Bitcoin.Features.Interop.Payloads
+{
+    [Payload("conversionrequeststate")]
```

Comment:
I think these names are limited to a much shorter string?

Reply:
Fixed